src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs

```diff
@@ -32,7 +32,7 @@ internal partial class BatchFixAllProvider : FixAllProvider
 
         #region "AbstractFixAllProvider methods"
 
-        public override async Task<CodeAction?> GetFixAsync(FixAllContext fixAllContext)
+        public sealed override async Task<CodeAction?> GetFixAsync(FixAllContext fixAllContext)
```

Comment:
i tried to seal a lot of our internal overrides.  much easier ot understand if we don't have to consider inheritance.